bin/dev

```diff
@@ -0,0 +1,16 @@
+#!/usr/bin/env sh
```

Comment:
ah, is this the command you're running now to run the server? 

I think it's a good idea! Just checking.

Reply:
![image](https://github.com/RossaMania/PushPin/assets/98660173/ab5cce68-5c86-4f5c-bd1f-06a208444df0)

Yes! The Procfile.dev and bin/dev files are installed automatically when Tailwind CSS is installed. This bin/dev file is similar to the "npm run dev" command in a React project. This should definitely come in handy when deploying the project to render.com. 

Once the server is running we should be able to start using Tailwind CSS in the application. I plan on making a homepage, and styling up the home page and trip page with Tailwind CSS. I figure I would keep it simple and start with the test boilerplate from the Tailwind docs.